cli/src/log.rs

```diff
@@ -25,6 +25,7 @@ pub fn init_logger(verbose: bool, quiet: bool, env_log_level: Result<String, Var
         };
         env_logger::builder()
             .filter_level(flag_log_level)
+            .format_level(false)
```

Comment:
Why do we not want to print the level?

Reply:
The `info` outputs get a `[Info]` bracket even though we want to use it as a normal log where the output shouldn't go to stdout

Reply:
I'm going to write a custom formatter for this to better match what we want